.gitignore

```diff
@@ -37,4 +37,6 @@ LanguageTool-5.4
 package-lock.json
 learning_observer/learning_observer/static_data/google/
 learning_observer/learning_observer/static_data/admins.yaml
-.ipynb_checkpoints/
\ No newline at end of file
+.ipynb_checkpoints/
+learning_observer/learning_observer/config.ini
```

Comment:
I don't think this is needed once the auth stuff is figured out.

Reply:
Yes, it's now removed